README.md

```diff
@@ -77,6 +77,34 @@ Schema::create('table', function (Blueprint $table) {
 });
 ```
 
+If you are want delete partial unique index, use this method:
```

Comment:
If you want to delete ...

or

To delete ...

Reply:
fixed